sympy/physics/units.py

```diff
@@ -270,7 +274,7 @@ def free_symbols(self):
 amu = amus = gram / avogadro / mol
 mmu = mmus = gram / mol
 quart = quarts = Rational(231, 4) * inch**3
-eV = 1.602176487e-19 * J
+eV = Float('1.6021766208','') * 10**-19 * J
```

Comment:
Since you aren't using Float anywhere else and other instances used `Rational(string)` let's remove (sorry!) the Float import and just write this as `eV = Rational('1.6021766208') * ten**-19 * J`.